assets/styles/scss/footer.scss

```diff
@@ -0,0 +1,80 @@
+@import "mixins.scss";
+@import "colors.scss";
+@import "style.scss";
+.fw-f-logo { 
+    position: relative;
+    right: 67px;
+}
+footer {
+    background-color: $bgcolorprimary;
+    padding: 5% 0%;
+}
+.fw-f-elements {
+    @include block60;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+}
+.fw-f-email {
+    color: $colortextsecondary;
+}
+.fw-f-address {
+    color: $colortextgrey;
+    font-family: 'Merriweather Sans';
+    padding-bottom: 0;
+  }
+.fw-f-contact-us {
+    @include flex-cols;
+    flex-grow: 4;
+    padding-right: 10%;
+  }
+input { 
+    @include in-t;
+}
+button {
+    @include bu-t;
```

Comment:
if a mixin is not going to be reused, it is better to avoid mixins and just put all related css here in class definition

Reply:
Resolved